/scratch/ptolemy/users/kg1623/projects/deep-learning/audio-classification-using-ViT/venv/lib/python3.10/site-packages/torch_audiomentations/core/transforms_interface.py:76: FutureWarning: Transforms now expect an `output_type` argument that currently defaults to 'tensor', will default to 'dict' in v0.12, and will be removed in v0.13. Make sure to update your code to something like:
  >>> augment = Gain(..., output_type='dict')
  >>> augmented_samples = augment(samples).samples
  warnings.warn(
/scratch/ptolemy/users/kg1623/projects/deep-learning/audio-classification-using-ViT/venv/lib/python3.10/site-packages/torch_audiomentations/core/transforms_interface.py:76: FutureWarning: Transforms now expect an `output_type` argument that currently defaults to 'tensor', will default to 'dict' in v0.12, and will be removed in v0.13. Make sure to update your code to something like:
  >>> augment = AddColoredNoise(..., output_type='dict')
  >>> augmented_samples = augment(samples).samples
  warnings.warn(
/scratch/ptolemy/users/kg1623/projects/deep-learning/audio-classification-using-ViT/venv/lib/python3.10/site-packages/torch_audiomentations/core/transforms_interface.py:76: FutureWarning: Transforms now expect an `output_type` argument that currently defaults to 'tensor', will default to 'dict' in v0.12, and will be removed in v0.13. Make sure to update your code to something like:
  >>> augment = PitchShift(..., output_type='dict')
  >>> augmented_samples = augment(samples).samples
  warnings.warn(
/scratch/ptolemy/users/kg1623/projects/deep-learning/audio-classification-using-ViT/venv/lib/python3.10/site-packages/torch_audiomentations/core/transforms_interface.py:76: FutureWarning: Transforms now expect an `output_type` argument that currently defaults to 'tensor', will default to 'dict' in v0.12, and will be removed in v0.13. Make sure to update your code to something like:
  >>> augment = Shift(..., output_type='dict')
  >>> augmented_samples = augment(samples).samples
  warnings.warn(
Using the latest cached version of the dataset since danavery/urbansound8K couldn't be found on the Hugging Face Hub
Found the latest cached dataset configuration 'default' at /scratch/ptolemy/users/kg1623/.cache/huggingface/datasets/danavery___urbansound8_k/default/0.0.0/8aa9177a0c5a6949ee4ee4b7fcabb01dfd4ae466 (last modified on Sat Apr 26 19:45:24 2025).
  0%|          | 0/100 [00:00<?, ?it/s]  1%|          | 1/100 [01:02<1:43:35, 62.79s/it]  2%|▏         | 2/100 [02:00<1:38:04, 60.05s/it]  3%|▎         | 3/100 [02:58<1:35:03, 58.80s/it]  4%|▍         | 4/100 [03:55<1:33:22, 58.36s/it]  5%|▌         | 5/100 [04:53<1:31:48, 57.99s/it]  6%|▌         | 6/100 [05:51<1:30:45, 57.93s/it]  7%|▋         | 7/100 [06:47<1:28:59, 57.41s/it]  8%|▊         | 8/100 [07:43<1:27:23, 57.00s/it]  9%|▉         | 9/100 [08:40<1:26:17, 56.90s/it] 10%|█         | 10/100 [09:37<1:25:32, 57.03s/it] 11%|█         | 11/100 [10:33<1:24:15, 56.80s/it] 12%|█▏        | 12/100 [11:32<1:23:56, 57.23s/it] 13%|█▎        | 13/100 [12:29<1:23:08, 57.34s/it] 14%|█▍        | 14/100 [13:26<1:21:56, 57.17s/it] 15%|█▌        | 15/100 [14:22<1:20:42, 56.97s/it] 16%|█▌        | 16/100 [15:19<1:19:48, 57.00s/it] 17%|█▋        | 17/100 [16:17<1:19:07, 57.19s/it] 18%|█▊        | 18/100 [17:14<1:18:07, 57.16s/it] 19%|█▉        | 19/100 [18:11<1:17:01, 57.06s/it] 20%|██        | 20/100 [19:08<1:15:52, 56.91s/it] 21%|██        | 21/100 [20:05<1:15:04, 57.01s/it] 22%|██▏       | 22/100 [21:02<1:14:04, 56.98s/it] 23%|██▎       | 23/100 [21:59<1:13:15, 57.08s/it] 24%|██▍       | 24/100 [22:57<1:12:33, 57.29s/it] 25%|██▌       | 25/100 [23:53<1:11:12, 56.97s/it] 26%|██▌       | 26/100 [24:51<1:10:39, 57.28s/it] 27%|██▋       | 27/100 [25:47<1:09:19, 56.98s/it] 28%|██▊       | 28/100 [26:44<1:08:21, 56.96s/it] 29%|██▉       | 29/100 [27:40<1:06:52, 56.51s/it] 30%|███       | 30/100 [28:37<1:06:22, 56.89s/it] 31%|███       | 31/100 [29:35<1:05:29, 56.95s/it] 32%|███▏      | 32/100 [30:32<1:04:39, 57.05s/it] 33%|███▎      | 33/100 [31:29<1:03:40, 57.02s/it] 33%|███▎      | 33/100 [32:24<1:05:48, 58.94s/it]
/scratch/ptolemy/users/kg1623/projects/deep-learning/audio-classification-using-ViT/venv/lib/python3.10/site-packages/torch/__init__.py:2150: TracerWarning: Converting a tensor to a Python boolean might cause the trace to be incorrect. We can't record the data flow of Python values, so this value will be treated as a constant in the future. This means that the trace might not generalize to other inputs!
  assert condition, message
